style.css

```
@@ -0,0 +1,11 @@
+#navbar
+{
+   position: fixed;
+   top: 0;
+}
```

🙌 I think it would be better if you remove unnecessary spaces to respect good practices and make your code more readable. 📑

```suggestion
#navbar {
   position: fixed;
   top: 0;
}
```